include/operations/blas_operators.hpp

```diff
@@ -40,22 +41,19 @@ namespace blas {
 template <typename ScalarT>
 struct IndVal {
   using value_type = ScalarT;
-  size_t ind;
-  value_type val;
+  size_t ind = UINT_MAX;
```

Comment:
For debug purposes, so it does  not spit an arbitrary value, but the one that is certainly wrong.